src/components/core/base/test/GeoValidatorHOC.spec.tsx

```diff
@@ -98,4 +98,40 @@ describe("GeoValidatorHOC", () => {
         expect(wrapper.find(ErrorComponent).exists()).toEqual(true);
         expect(onError).toBeCalledTimes(2);
     });
+
+    it("should update props when filters changed", () => {
```

Comment:
I prefer to `should component be updated to new prop when filters changed`

Reply:
updated.
Thanks :)